src/cpu/aarch64/jit_generator.hpp

```diff
@@ -211,6 +211,21 @@ class jit_generator : public Xbyak_aarch64::CodeGenerator, public c_compatible {
         L(label);
     }
 
+    template <typename T>
+    Xbyak_aarch64::XReg EVEX_compress_addr(const Xbyak_aarch64::XReg &addr,
```

Comment:
My understanding is EVEX relates to x64 only. Could you issue a comment why it's kept this way, please?

Reply:
Removed this function.